rmgpy/statmech/rotationTest.py

```diff
@@ -108,10 +108,10 @@ def test_getPartitionFunction_classical(self):
         """
         self.mode.quantum = False
         Tlist = numpy.array([300, 500, 1000, 1500, 2000])
-        Qexplist = numpy.array([72.6691, 121.115, 242.230, 363.346, 484.461])
-        for T, Qexp in zip(Tlist, Qexplist):
-            Qact = self.mode.getPartitionFunction(T)
-            self.assertAlmostEqual(Qexp, Qact, delta=1e-4*Qexp)
+        q_exp_list = numpy.array([72.6691, 121.115, 242.230, 363.346, 484.461])
+        for T, q_exp in zip(Tlist, q_exp_list):
```

Comment:
maybe `temperature` instead of `T`? (we don't want to use `t` either)
Same for the lines below